pkg/cloud/aws/actuators/machine_scope.go

```diff
@@ -57,7 +64,7 @@ func NewMachineScope(params MachineScopeParams) (*MachineScope, error) {
 	if params.Client != nil {
 		machineClient = params.Client.Machines(params.Machine.Namespace)
 	}
-
+	scope.Logger = scope.Logger.WithName(params.Machine.Name)
```

Comment:
Should this be: `scope.Logger.WithName(params.Cluster.APIVersion).WithName(params.Cluster.Namespace).WithName(params.Cluster.Name).WithName(params.Machine.Name)` instead to align with the cluster scope logger and also capture the cluster context?

Reply:
No, the call to NewScope above on line 44 will create that namespaced context and then this line will append the machine name to the end of it. So at this point it's already got APIVersion, Namespace, Name as part of the context